src/pages/help-centre/_utility.ts

```diff
@@ -22,7 +22,10 @@ export const splitArticles = (array: ArcticlesType[], length: number): string[]
         return result
     }, [])
 
-export const euArticles = (array: string[]): [string, string[], string[], string[]] => {
-    const second_array = [...array[1], array[2][0]]
-    return [array[0], second_array, [array[2][1]], [array[2][2]]]
+export const euArticles = (array: string[]): [string, string[], string[]] => {
+    const second_array = [
+        ...array[1].filter((el) => el.category.props.translate_text !== 'Deriv X'),
```

Comment:
Can we have an array for the items we should hide for EU clients? So later it will be easy for us to add an exlusion.

Reply:
Good point. Done